packages/expo-cellular/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Fix null cellular information on iOS. ([#12710](https://github.com/expo/expo/pull/12710) by [@randomhajile](https://github.com/randomhajile))
 - Enable kotlin in all modules. ([#12716](https://github.com/expo/expo/pull/12716) by [@wschurman](https://github.com/wschurman))
+- Add TypeScript type to the exported constants. ([#12838](https://github.com/expo/expo/pull/12838) by [@simek](https://github.com/simek))
```

Comment:
What specific type are you referring to? 🤔

Reply:
https://github.com/expo/expo/pull/12838/files#diff-9aac863a23b1c60e6b4cf38ccf0979d621ea4c8dac127f10c0b2e5d78b5d526eL8-L12

Reply:
```suggestion
- Add TypeScript types to the exported constants :`allowsVoip`, `carrier`, `isoCountryCode`, `mobileCountryCode` and `mobileNetworkCode`. ([#12838](https://github.com/expo/expo/pull/12838) by [@simek](https://github.com/simek))
```